### PR TITLE
[IMPROVE] Omnichannel Visitor Endpoints error handling

### PR DESCRIPTION
**Pull request by KevLehman**

### CODE REVIEW

File: app/livechat/server/api/v1/visitor.ts [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+			callId: String,
+		});
+
+		const { token, callStatus, rid, callId } = this.bodyParams;
+		const guest = findGuest(token);
+		if (!guest) {
+			throw new Meteor.Error('invalid-token');
+		}
+		const status = callStatus;
+		Livechat.updateCallStatus(callId, rid, status, guest);
```
Comment (murtaza98) [inline app/livechat/server/api/v1/visitor.ts:157]:
```suggestion
		Livechat.updateCallStatus(callId, rid, callStatus, guest);
=
```

File: app/livechat/server/api/v1/visitor.ts [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+			rid: String,
+			callId: String,
+		});
+
+		const { token, callStatus, rid, callId } = this.bodyParams;
+		const guest = findGuest(token);
+		if (!guest) {
+			throw new Meteor.Error('invalid-token');
+		}
+		const status = callStatus;
```
Comment (murtaza98) [inline app/livechat/server/api/v1/visitor.ts:156]:
```suggestion
```

Comment (murtaza98) [reply app/livechat/server/api/v1/visitor.ts:156]: This variable assignment seems redundent

File: app/livechat/server/api/v1/visitor.ts [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+			callId: String,
+		});
+
+		const { token, callStatus, rid, callId } = this.bodyParams;
+		const guest = findGuest(token);
+		if (!guest) {
+			throw new Meteor.Error('invalid-token');
+		}
+		Livechat.updateCallStatus(callId, rid, callStatus, guest);
+=
```
Comment (KevLehman) [inline app/livechat/server/api/v1/visitor.ts:157]:
```suggestion
```

**Pull request merged**